test/src/main/java/org/apache/accumulo/test/ConditionalWriterIT.java

```diff
@@ -883,15 +884,19 @@ public void testBigBatch() throws Exception {
 
         int count = 0;
 
-        // TODO check got each row back
+        Set<String> rowsReceived = new HashSet<>();
         while (results.hasNext()) {
           Result result = results.next();
+          rowsReceived.add(new String(result.getMutation().getRow()));
```

Comment:
Should explicitly use `UTF_8` when constructing a String from byte array, otherwise, this could fail on some platforms where `UTF_8` is not the default.